docs/combine-active-promotions.md

```diff
@@ -0,0 +1,33 @@
+# Combine Active Promotions
+
+VirtoCommerce Marketing Module represents two basic promotion combination policies:  
+
+1. **Best reward policy** - select the promotion reward, that most of all promotions benefits the customer.  
```

Comment:
Maybe better 
 - select the promotion reward that benefits a customer the most.

Reply:
OK